test/unit/karma.conf.js

```diff
@@ -0,0 +1,36 @@
+// This is a karma config file. For more details see
+//   http://karma-runner.github.io/0.13/config/configuration-file.html
+// we are also using it with karma-webpack
+//   https://github.com/webpack/karma-webpack
+
+var webpackConfig = require('../../build/webpack.test.conf');
+
+module.exports = function (config) {
+  config.set({
+    // to run in additional browsers:
+    // 1. install corresponding karma launcher
+    //    http://karma-runner.github.io/0.13/config/browsers.html
+    // 2. add it to the `browsers` array below.
+    browsers: ['PhantomJS'],
+    frameworks: ['mocha', 'sinon-chai', 'phantomjs-shim'],
+    reporters: ['spec', 'coverage'],
+    files: [
+      '../../node_modules/es6-promise/dist/es6-promise.auto.js',
```

Comment:
There was a comment on the detailed commit message 😄
 
I was trying to run the unit tests and got this error:

<img width="472" alt="screen shot 2017-08-14 at 14 21 53" src="https://user-images.githubusercontent.com/8316698/29269784-165d0d00-80fc-11e7-8175-6ebae0ccc0b3.png">

Searching for an answer, found this:
https://github.com/vuejs-templates/webpack/issues/474